### DISCUSSION TITLE
$(brew --repository homebrew/cask)/developer/bin

### DISCUSSION THREAD

**Discussion by dronenb** [Homebrew/discussions#5126] [Writing Formulae/Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.8-7-ga783aaf
ORIGIN: https://github.com/Homebrew/brew
HEAD: a783aaf03f83d4d9ef5802a4900bac797a0659fc
Last commit: 3 hours ago
Core tap JSON: 12 Feb 23:08 UTC
Core cask tap HEAD: 5c3611bffb7721eee15559b4ed8bfeacd575dafc
Core cask tap last commit: 64 minutes ago
Core cask tap JSON: 12 Feb 23:18 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: ["--no-quarantine"]
HOMEBREW_DISPLAY: :0
HOMEBREW_EDITOR: /opt/homebrew/bin/vim
HOMEBREW_MAKE_JOBS: 10
HOMEBREW_NO_ENV_HINTS: set
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.3.1-arm64
CLT: 15.1.0.0.1.1700200546
Xcode: 15.2
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Since `homebrew/cask` is no longer tapped by default, is the expectation that people developing casks must tap that repo so that they can use the tools discussed [here](https://docs.brew.sh/Cask-Cookbook#uninstall-pkgutil), despite it not being recommended?

**Comment by p-linnane** [comment, marked as answer]
Yes, that's correct. It's not recommended to have it tapped _for most users_ now that we install from API by default. Developing requires you to have a copy of the repo in order to test and open a PR.

**Reply by dronenb** [reply]
Thanks @p-linnane, that answer seems obvious to me now that you've said it. I have been mostly writing casks in my own repo for older software that doesn't belong upstream so it didn't jump out to me...
